data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Nou, blijkbaar heb je het niet gemist. @Corsleutel had hier het volgende over te vertellen:

> In het geval van 'achtbaanrails' en ook bijvoorbeeld 'treinrails', heeft 1 spoor altijd 2 rails die samen 1 spoor maken, derhalve is een achtbaanrails gewoon correct. 'een treinspoor = een treinrails'

Een spoorrail is letterlijk een spoorstaaf. Ik mag hopen dat de treinen van de NS gewoon op twee spoorstaven rijden... (Je zegt toch ook niet een spoorstaven, omdat ze beide onderdeel zijn van het zelfde spoor? :confused:) Oftewel, rails is gewoon meervoud.